xml/System/Span`1.xml

```diff
@@ -295,10 +341,15 @@
         <Parameter Name="obj" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="obj">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="obj">Not supported.</param>
+        <summary>Calls to this method are not supported.</summary>
```

Comment:
Should this match the documentation of the GetHashCode method? Both throw NotSupportedException.

Reply:
I got a compiler error instead, @ahsonkhan. Is that expected?

Reply:
Ah yes, maybe that's worth mentioning as well. Since span is a ref struct, it cannot be boxed and hence cannot be converted to object. So, the compiler will return such an error if you try to pass in a span to any method that expects an object.